examples/gauge/derived-gauge-quickstart.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This is an example shows how to create a Derived Gauge metric. This gauge is
+ * self sufficient once created
+ */
+
+const { Metrics, MeasureUnit } = require('@opencensus/core');
+
+// [UNCOMMENT THIS BLOCK to visualize the data =======================]
+// // Enable OpenCensus exporters to export gauges to Stackdriver Monitoring.
+// const { StackdriverStatsExporter } = require('@opencensus/exporter-stackdriver');
+// const exporter = new StackdriverStatsExporter({ projectId: 'projectId' });
+// const { globalStats } = require('@opencensus/core');
+// globalStats.registerExporter(exporter);
+// [END setup_exporter ==============================================]
+
+setTimeout(() => {
+  console.log('Timeout executed.');
```

Comment:
Could you document what this timeout is for? I remember in another example the timeout kept the Node process alive while the metric was being collected.